torch/fx/passes/fake_tensor_prop.py

```diff
@@ -20,16 +22,20 @@ class FakeTensorProp(torch.fx.Interpreter):
     Args:
          module (GraphModule): The module to be executed
          mode (Optional[FakeTensorMode]): The dispatch mode used to execute computation indicated by each FX Node.
+         override (bool): Whether to override the `val` attribute of each Node with the fake tensor result.
+             If False, only the Nodes without `val` will be assigned the fake tensor result.
     """
-    def __init__(self, module: torch.fx.GraphModule, mode: Optional[FakeTensorMode] = None):
+    def __init__(self, module: torch.fx.GraphModule, mode: Optional[FakeTensorMode] = None, override: bool = True):
```

Comment:
```suggestion
    def __init__(self, module: torch.fx.GraphModule, mode: Optional[FakeTensorMode] = None, override: Optional[bool]= True):
```

Reply:
`Optional[bool]` would mean we expect None as an input. Was this intended?

Reply:
I prefer this to be default True. Optional to None doesn't seen to be a good idea? Because I would like to keep it's default behavior.